src/connection.rs

```diff
@@ -60,12 +71,15 @@ impl Connection {
         connection
     }
 
-    /// Connect to an AMQP Server
+    /// Connect to an AMQP Server.
+    ///
+    /// The URI must be in the format `amqp://127.0.0.1:5672/%2f`,
+    /// where the default virtual host (`/`) has to be ecaped to URL encoding.
```

Comment:
```suggestion
    /// where the default virtual host (`/`) has to be escaped to URL encoding.
```

Reply:
Also, this is a bit tricky but:
- `amqp://127.0.0.1:5672/%2f` means vhost = "/" since %2f is /
- `amqp://127.0.0.1:5672/` means vhost = "" since the "path" part is empty
- `amqp://127.0.0.1:5672` means vhost = "/" since the "path" part is omitted so we use the default